CMakeSettings.json

```diff
@@ -0,0 +1,42 @@
+﻿{
```

Comment:
I think this file needs to go, it's setup specific

Reply:
You could make it empty strings sure, or put them on ../ like i do with linux, but they are cache variables, so after first generation you should be able to just set them, that's why they're predefined so they show up in the mneu.

Reply:
The paths are now generic `../`